src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -0,0 +1,115 @@
+    
+    <!-- Hook the target that fetches the additional files into to the build process. 
+         This needs to be set after the SDK targets have been imported.
+    -->
+    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);GetTypeScriptFiles</PrepareForBuildDependsOn>
+    <!-- By default the SDK targets only zip Release builds. We'll always zip since the file is very large otherwise. -->
+    <ZipPackageCompressionLevel>Normal</ZipPackageCompressionLevel>
+  </PropertyGroup>
+  
+  <ItemGroup Label="GetTypScriptFiles inputs and outputs">
+    <!-- Specifies the target inputs and outputs so MSBuild can skip fetching the files if the analyzer version has not changed. -->
+    <AdditionalFilesInput Include="..\EmbeddedSonarAnalyzer.props" />
+    <AdditionalFilesOutput Include="$(OutputPath)$(AssemblyName).vsix" />
+  </ItemGroup>
+  
+  <Target Name="GetTypeScriptFiles" Inputs="@(AdditionalFilesInput)" Outputs="@(AdditionalFilesOutput)" >
+    <Message Importance="high" Text="Fetching the TypeScript files..." />
+    <!-- Note: building the helper project explicitly using MSBuild rather than referencing it via a ProjectReference.
+         This makes is easier for us to control when the build is performed, so we can trigger it early enough
+         to include the downloaded files as normal Content files in this project.
+         Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
+         DownloadCFamilyPlugin.csproj to avoid getting file-locking errors when rebuilding.
+    -->
+    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadCFamilyPlugin\DownloadCFamilyPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(TypeScriptTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedTypeScriptFiles" LogStandardErrorAsError="true" IgnoreExitCode="false" />
+    <ItemGroup>
+      <!-- Note (1): vsixutil.exe will fail with an OutOfMemory error if there are too many files in the VSIX (the size of the files
+                     doesn't matter, it's the number that counts.
+		     So, we are excluding some files that are not required i.e. the *.map files.
+		     If we need to exclude more files then other candidates are:
+		     * the localization files for the TypeScript compiler
+		     * some modules include "test" folders which may not be needed
+		     * *.d.ts files i.e. type definition files. Ones in modules other than the TypeScript compiler/eslint modules *might* not be needed
+      -->
+      <!-- Note (2): the file name has to be relative. The "Include" value is used to set the value of the "vsixSubPath" metadata item.
+			If this is an absolute path the VSSDK tools won't complain, but the file won't be included in the vsix. -->
+      <Content Include="$(TypeScriptFolderName)\**\*"
+               Exclude="$(TypeScriptFolderName)\**\*.map" >
+        <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```

Comment:
Setting this to `Never` doesn't seem to make any difference - I still seem to end up with the files in the output directory, and the copied to the Integration.Vsix output directory. Not sure why, but it doesn't cause a problem other than potentially slowing the build, so it's not urgent.